package.json

```diff
@@ -18,7 +18,7 @@
         "prettier": "prettier --write .",
         "prettier:check": "prettier --check .",
         "prepublishOnly": "yarn test && yarn build",
-        "task:db-init": "ts-node ./tasks/db-init.ts"
+        "task:db-init": "NODE_ENV=test ts-node ./tasks/db-init.ts"
```

Comment:
Why mandatory NODE_ENV=test?

Reply:
Hmm... I basically put it so I wouldn't wipe my localhost `posthog` database, but would instead create tables in the `posthog_test` database. This just made running the script a bit easier on the command line.

The only time you would ever run the `create table` command within `posthog-plugin-server` is in a test environment. Otherwise all of this is handled by django. So as an added security I think it's fine... 🤷

Reply:
Ah, makes sense, the script could maybe be named to indicate that it's only for testing.

Reply:
sure, renamed to `init-test-db.ts` and fixed the merge conflicts